ang/civicase/Search.html

```diff
@@ -2,77 +2,69 @@
+        <civicase-ui-date-range ng-if="isEnabled('start_date')" date-range="filters.start_date"></civicase-ui-date-range>
       </div>
-    </div>
-    <div class="form-group">
-      <label ng-if="isEnabled('end_date')" class="control-label col-lg-2">{{ ts('Case End Date') }}</label>
-      <div class="col-lg-4">
-        <div ng-if="isEnabled('end_date')" crm-ui-date-range="filters.end_date"></div>
+      <div class="civicase__case-filter-form-elements">
+        <label ng-if="isEnabled('end_date')">{{ ts('Case End Date') }}</label>
+        <civicase-ui-date-range ng-if="isEnabled('end_date')" date-range="filters.end_date"></civicase-ui-date-range>
       </div>
-    </div>
-    <div class="form-group">
-      <label class="control-label col-lg-2">{{ ts('Tagged') }}</label>
-      <div class="col-lg-4">
-        <input class="form-control" ng-list crm-entityref="{entity: 'Tag', api: {params: {used_for: {'LIKE':'%civicrm_case%'}, is_selectable: 1}}, select: {multiple: true}}" ng-model="filters.tag_id">
+      <div class="civicase__case-filter-form-elements">
+        <label>{{ ts('Tagged') }}</label>
+        <div>
+          <input class="form-control" ng-list crm-entityref="{entity: 'Tag', api: {params: {used_for: {'LIKE':'%civicrm_case%'}, is_selectable: 1}}, select: {multiple: true}}" ng-model="filters.tag_id">
+        </div>
       </div>
-    </div>
-    <div ng-include="'~/civicase/CaseSearchCustom.html'"></div>
-    <hr />
-    <div>
-      <button class="btn btn-primary btn-sm civicase__case-filter-panel__button civicase__case-filter-panel__button--search" ng-click="doSearch()" ng-show="expanded">
-        <i class="fa fa-search"></i> {{ ts('Search') }}
-      </button>
+      <div class="civicase__case-filter-form-elements" ng-include="'~/civicase/CaseSearchCustom.html'"></div>
     </div>
   </form>
   <div class="panel-body civicase__case-filter-panel__description" ng-if="!expanded && filterDescription.length">
-    <button class="btn btn-danger btn-sm pull-right civicase__case-filter-panel__button civicase__case-filter-panel__button--clear" ng-click="clearSearch()">
-      <i class="fa fa-ban"></i> {{ ts('Clear Search') }}
-    </button>
-    <ul class="civicase__filter-search-description">
-      <li ng-repeat="des in filterDescription" class="col-lg-6">
+    <ul class="civicase__filter-search-description-list-container">
+      <li ng-repeat="des in filterDescription" class="col-lg-6 civicase__filter-search-description-list">
         <strong>{{ des.label }}:</strong> {{ des.text }}
       </li>
     </ul>
+    <button class="btn btn-danger btn-sm pull-right civicase__case-filter-panel__button" ng-click="clearSearch()">
+      <i class="fa fa-ban"></i> {{ ts('Clear Search') }}
+    </button>    
   </div>
 </div>
+
+  <!-- TOdo the logic to be used while implemeting the case manage dropdown  -->
+  <!-- <div class="form-group" ng-if="isEnabled('case_manager')"> -->
```

Comment:
This will be the part of the work to be done in C51-35. This form element is being implemented as the third dropdown. I will be working on that ticket after this.